king_phisher/client/dialogs/configuration.py

```diff
@@ -32,7 +32,10 @@
 
 import collections
 import logging
+import os
 import string
+import urllib.parse
+
```

Comment:
Don't need a second blank line here per the [Style Guide](https://king-phisher.readthedocs.io/en/latest/development/style_guide.html#style-guide).